src/scripts/testbot/TESTBOT_RESPOND_PROMPT.md

```
@@ -0,0 +1,51 @@
+      "reply": "Renamed the describe block to match the function name. Also added an edge case test for empty input."
+    },
+    {
+      "comment_id": "3066590421",
+      "reply": "Removed the redundant assertion. The behavior is already covered by test_parse_basic."
+    }
+  ]
+}
+```
+
+**Fields:**
+- `commit_message`: A concise summary prefixed with `testbot: ` (subject under 72 chars).
```

markdownlint-cli2 (0.22.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)